src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static final String ADD_TASK_MSG = "Got it. I've added this task: ";
+    private static final String ERROR_MSG = "Sorry, I didn't quite get what you want me to do :(";
+    private static final int MAX_TASKS = 100;
+
     public static void main(String[] args) {
+        showHelloGreeting();
+        executeResponses();
```

Comment:
Perhaps this method name can be changed to something more precise and clearer to the user?